package.json

```diff
@@ -51,10 +51,12 @@
     "eslint": "^1.10.3",
     "eslint-config-kentcdodds": "^5.0.0",
     "eslint-plugin-mocha": "^1.1.0",
+    "execa": "^0.2.2",
```

Comment:
Hum... true. It is a bit weird to think about where things go in this situation. :confounded:
Although when running `npm install ghooks` on a project, the `dist` directory should already be there, as that's what we publish to NPM, so in theory only `path-exists` would be required as a regular dependency. I guess we can play safe and have them both as regular deps, though.

Thanks! We good after these changes?